src/components/BaseComponents/BaseScreen.tsx

```diff
@@ -8,7 +8,7 @@ const styles = () =>
       top: '85px',
       bottom: '56px',
       width: '100%',
-      height: 'auto',
+      height: '70%',
```

Comment:
I think changing this line from `auto` makes some of the UI on other screens look weird. The scroll is a problem on other components as well and is an issue Annie has to deal with. She probably has some advice (I think we might be able to just add padding / margin to the bottom of the list so that it doesn't clip?

Link to conversation about whitespace: https://github.com/calblueprint/meepanyar/pull/87#issuecomment-820041876

cc @wangannie is there an easy fix you'd recommend for the last CustomerCard clipping?

Screenshot of funky UI bc of this change:
![image](https://user-images.githubusercontent.com/35501399/114954833-36bdb600-9e10-11eb-803c-bbad60a851c5.png)

Reply:
Yeah definitely revert back to auto for now. I tried messing with this to find a workaround but I haven't gotten very far. I'll let you know if I come up with something. For now, just stick to adding extra padding or a blank div at the end of the list.